### PR TITLE
Update turtle_offspring_nesting.json

### PR DESCRIPTION
**Pull request by ss-is-master-chief**
@henrykironde Please have a look

### CODE REVIEW

File: scripts/turtle_offspring_nesting.json [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
-    "description": "",
+    "citation": "Lisa E. Schwanz, Rachel M. Bowden, Ricky-John Spencer, and Fredric J. Janzen. 2009. Nesting ecology and offspring recruitment in a long-lived turtle. Ecology 90:1709. [https://doi.org/10.6084/m9.figshare.3531323.v1]",
+    "description": "Valuable empirical resource for exploring important facets of nesting ecology and hatchling recruitment in a wild population of a long-lived species.",
     "homepage": "https://figshare.com/articles/Data_Paper_Data_Paper/3531323",
     "keywords": [
-        "Taxon > Animals"
+        "Taxon > Animals",
+        "demography",
+        "early-life-stages",
+        "fluctuating-environments",
```
Comment (henrykironde) [inline scripts/turtle_offspring_nesting.json]: Have you tried to test the changes using the data retriever list by keyword command? 

And how about if we remove  `-` and just provide strings
`"demography", "early life stages", "fluctuating environments",......`

**Comment by henrykironde** [comment]
What are some of the commands  you have used to test this PR?

**Comment by ss-is-master-chief** [comment]
@henrykironde  I have tested it through the python interface. Removed `-`. Please advice. 
```
for data in rt.datasets():
    if data.name=="turtle-offspring-nesting":
        for key in data.keywords:
            print(key)
        print(data.version)
        print(data.citation)
```

Which gave me : 

```
Taxon > Animals
demography
early life stages
fluctuating environments
maternal effects
nesting behavior
predation
temperature dependent sex determination
turtles
1.0.1
Lisa E. Schwanz, Rachel M. Bowden, Ricky-John Spencer, and Fredric J. Janzen. 2009. Nesting ecology and offspring recruitment in a long-lived turtle. Ecology 90:1709.```

**Comment by ss-is-master-chief** [comment]
I'll push the updated `version.txt` file in some time

**Comment by henrykironde** [comment]
@ss-is-master-chief update the version.txt and this will be ready

**Comment by ss-is-master-chief** [comment]
screen shot 2018-02-22 at 1 04 20 am

screen shot 2018-02-22 at 1 03 00 am

@henrykironde Please advice

**Comment by ss-is-master-chief** [comment]
The above `prompt` doesn't occur when I change the version number to `1.0.2` but the test fails.

**Comment by ss-is-master-chief** [comment]
I'll close this for now.. Will open it ASAP

**Comment by henrykironde** [comment]
In the retriever we have some git hooks that check if the version is updated. In your case, I think you are behind master so thats why its failing. You can force a commit by skipping the git hook. `git commit -m"message" -n`.  You should rebase your branch to the latest master

**Comment by ss-is-master-chief** [comment]
@henrykironde Thank you Sir. I have updated and pushed `version.txt`. Please check

File: scripts/turtle_offspring_nesting.json (hunk 1 above)
Comment (henrykironde) [reply scripts/turtle_offspring_nesting.json]: @ss-is-master-chief 
    "keywords": [
        "Taxon > Animals",
        "demography",
        "early life stages",
        "fluctuating environments",
        "maternal effects",
        "nesting behavior",
        "predation",
        "temperature dependent sex determination",
        "turtles"

    ],

**Comment by henrykironde** [comment]
Some changes in the review still not revised :)

**Comment by ss-is-master-chief** [comment]
@henrykironde Revised. Please have a look.

**Comment by zhangcandrew** [comment]
Hey @ss-is-master-chief, 

Looks good! 

Do you think you could squash your commits into 1 commit?

**Pull request merged**